src/example-relay/__github__/jest.config.js

```diff
@@ -0,0 +1,6 @@
+// @flow strict
+
+module.exports = {
+  rootDir: __dirname,
+  testMatch: ['<rootDir>/**/__tests__/**/?(*.)+(spec|test).js?(x)'],
+};
```

Comment:
Why do you need to replace this file? Could we just modify the currently existing one?

Reply:
original one uses `src/example-relay/scripts/jest/custom-transformer.js` -> shouldn't we strip this custom transformer as well?

Reply:
```
// This configuration is merged with the project configuration defined
// in this monorepo root.
module.exports = {
  rootDir: __dirname,
  transform: {
    '^.+\\.js$': '<rootDir>/scripts/jest/custom-transformer.js',
  },
};
```

vs 

```
module.exports = {
  rootDir: __dirname,
  testMatch: ['<rootDir>/**/__tests__/**/?(*.)+(spec|test).js?(x)'],
};
```

Reply:
OK, I will leave it up to you. It could be easier to leave there even the transformer if it doesn't break anything for OSS.

Reply:
thanks, it breaks because it tries to find `.babelrc.js` which is gone

Reply:
This can actually be easily fixed - check Shipit docs and search for `@x-shipit-enable` comments.

Reply:
That's also a way how to enable/disable some lines for `jest.config.js` (as an alternative approach).

Reply:
hmm, ok, I considered two options

- having one jest file an use `@x-shipit-enable`
- having two files for jest

I went for second one because this transform is not needed and require in that file produces another flow issue which would require me to put additional suppress comment into flowconfig